src/Microsoft.VisualStudio.Services.Agent/JobServerQueue.cs

```diff
@@ -311,7 +311,7 @@ private async Task ProcessWebConsoleLinesQueueAsync(bool runOnce = false)
                         {
                             Trace.Info($"Skip {batchedLines.Count - 2} batches web console lines for last run");
                             batchedLines = batchedLines.TakeLast(2).ToList();
-                            batchedLines[0].Insert(0, "...");
```

Comment:
Do we need this ? We could do something like batchesLines[0].Insert(0, new TimelineRecordLogLine("...", batchedLines[0][0].LineNumber - 1);

Reply:
remove this.

Reply:
remove it. :D